README.md

```diff
@@ -0,0 +1,31 @@
+Kuali Rice
+==========
+Kuali Rice is an application development framework and middleware suite developed for and by the higher education community.  Kuali Rice is being used by several Kuali universiy enterprise applications, for its build in eDocLite document routing, and as basis for custom applications.
```

Comment:
"for its build in eDocLite document routing"  is confusing to me.  Do you mean  "to build eDocLite documents"  or "for building eDocLite documents"  ?